packages/grafana-ui/src/components/TableInputCSV/TableInputCSV.tsx

```diff
@@ -59,19 +70,20 @@ export class TableInputCSV extends React.PureComponent<Props, State> {
   };
 
   render() {
-    const { width, height } = this.props;
+    const { width, height, theme } = this.props;
     const { data } = this.state;
+    const styles = getStyles(theme);
     return (
-      <div className="gf-table-input-csv">
+      <div className={styles.tableInputCsv}>
         <textarea
           style={{ width, height }}
           placeholder="Enter CSV here..."
           value={this.state.text}
           onChange={this.onTextChange}
-          className="gf-form-input"
+          className={cx(styles.textarea, 'gf-form-input')}
```

Comment:
Aha now I see what is going on, I think maybe you should use the `TextArea` component from grafana-ui and then you don't need to combine the `gf-form-input` style?

Reply:
Yeah...I think that would work perfectly fine